eng/tools/versioning/sync-constants.js

```diff
@@ -0,0 +1,65 @@
+let argv = require("yargs")
```

Comment:
Did you plan to keep this script as well? It looks like you also have it implemented in updatePackageConstants which is where I expected to see it.